src/hotspot/cpu/x86/interp_masm_x86.cpp

```diff
@@ -605,6 +605,10 @@ void InterpreterMacroAssembler::push_i(Register r) {
   push(r);
 }
 
+void InterpreterMacroAssembler::push_i_or_ptr(Register r) {
+  push_i(r);
```

Comment:
Should be `push(r)`: it is both cleaner and avoids a middle call to `push_i(r)`.